hypha/apply/funds/differ.py

```diff
@@ -18,11 +18,10 @@ def wrap_added(text):
 
 def compare(answer_a, answer_b, should_bleach=True):
     if should_bleach:
```

Comment:
Rename `should_bleach` to something like `should_clean`?

Reply:
yeah that makes more sense. I also added a docstring to that function while I was there.